### PR TITLE
Added endpoint for delete users sessions

### PR DESCRIPTION
**Pull request by RomanGod6**
## Changes
* Added a new endpoint for deleting a users session via there ID

### CODE REVIEW

**Comment by equalsgibson** [comment]
Looks good to me - could you please add a unit test for this new func, that shows the expected response from Zendesk (with dummy data in place of any actual usernames or ids?)

File: zendesk/support_users.go [hunk 1]
```diff
@@ -362,3 +362,26 @@ func (s UserService) Update(
...
+	if err != nil {
+		return UserResponse{}, err
+	}
+
+	if err := s.client.ZendeskRequest(request, &target); err != nil {
+		return UserResponse{}, err
+	}
+
+	return target, nil
+}
```
Comment (equalsgibson) [inline zendesk/support_users.go:387]: Can we move this out of the "User Service" and put it into the "Account Configuration" service?

File: zendesk/account_configuration_delete_sessions.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	"fmt"
+	"net/http"
+)
+
+// https://developer.zendesk.com/api-reference/ticketing/account-configuration/sessions
+type SessionService struct {
+	client *client
+}
+
+// https://developer.zendesk.com/api-reference/ticketing/account-configuration/sessions/#delete-session
```
Comment (aaronellington) [inline zendesk/account_configuration_delete_sessions.go:14]: this is the correct comment/link? Aren't you calling the bulk delete endpoint?

File: zendesk/account_configuration_delete_sessions.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+func (s SessionService) DeleteSession(
```
Comment (aaronellington) [inline zendesk/account_configuration_delete_sessions.go:15]:
```suggestion
func (s SessionService) BulkDelete(
```

**Review by aaronellington — CHANGES_REQUESTED** [review]

**Pull request merged**